caduceus.yaml

```
@@ -343,4 +330,4 @@
   # perSourceIncomingPayloadSizeBuckets:
   #   - 100
   #   - 1000
-  #   - 10000
+  #   - 10000
```

```suggestion
  #   - 10000
  
```
unix text file newline at eof